src/components/Animal.js

```diff
@@ -100,27 +127,28 @@ const Animal = ({ animalsDb, onAddAnimal, onUpdateAnimal, onDeleteAnimal }) => {
     // On click submit
     const onSubmit = (e) => {
         e.preventDefault()
-
-        if (!type && !breed && !disposition && 
-                !picture &&!availability && 
-                !newsItem && !description) {
-            return
-        }
-
-        const body = {
-            "type": type,
-            "breed": breed,
-            "disposition": disposition,
-            "picture": picture,
-            "availability": availability,
-            "newsItem": newsItem,
-            "description": description
-        }
+        
+        let body = {}
 
         // Update if id
         if (id) {
-            body["animal_id"] = id
+            if (breed.length === 0) {
+                setError(true)
+                return
+            }
 
+            for (const key in idAnimal) {
+                if (key === 'breeds' || key === 'disposition') {
+                    if (objectMapping[key].length > 0) {
+                        body[key] = objectMapping[key]
+                    }
+                } else if (objectMapping[key]) {
+                    body[key] = objectMapping[key]
+                } else {
+                    body[key] = idAnimal[key]
+                }
+            }
+            
             console.log(body)
```

Comment:
Did you want to leave the console.log in for production?

Reply:
This is just the setup with the dummy data for now, I decided to console.log the data instead. I can remove if that's preferred.

Reply:
I decided to comment it out instead.